aggregator.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+class Aggregator {
```

Comment:
I would prefer, a name like "presenter" here, because this is the layer that is concerned with taking the raw audit data and formatting it for human readable display. 

I totally get that it is aggregating multiple audits together however it is also doing another "arguably" more important step, "presenting" the data.